consensus/hotstuff/integration/instance_test.go

```diff
@@ -216,15 +217,21 @@ func NewInstance(t require.TestingT, options ...Option) *Instance {
 	)
 	in.signer.On("CreateQC", mock.Anything).Return(
 		func(votes []*model.Vote) *flow.QuorumCertificate {
-			voterIDs := make([]flow.Identifier, 0, len(votes))
+			voterIDs := make(flow.IdentifierList, 0, len(votes))
 			for _, vote := range votes {
 				voterIDs = append(voterIDs, vote.SignerID)
 			}
+
+			signerIndices, err := packer.EncodeSignerIdentifiersToIndices(in.participants.NodeIDs(), voterIDs)
+			if err != nil {
+				panic(fmt.Errorf("could not encode signer indices: %w", err))
```

Comment:
I think this panic just makes things more complicated, can we stick to require.NoError?

Reply:
[Fixed here](https://github.com/onflow/flow-go/pull/2140/files#diff-3aaac0dad8c18f64b4dea3aa6b2ea3dc9330028ca6cb5d4c56887b25491165d8)